applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -77,25 +79,29 @@ void DistanceModificationProcess::ExecuteInitialize() {
     KRATOS_CHECK_VARIABLE_IN_NODAL_DATA(NODAL_H, r_node);
     KRATOS_CHECK_VARIABLE_IN_NODAL_DATA(DISTANCE, r_node);
 
-    // Obtain NODAL_H values
-    FindNodalHProcess NodalHCalculator(mrModelPart);
-    NodalHCalculator.Execute();
-
     KRATOS_CATCH("");
 }
 
 void DistanceModificationProcess::ExecuteBeforeSolutionLoop() {
 
     KRATOS_TRY;
 
+    // Initialize the number of bad cuts to a non-zero value
```

Comment:
very minor: you could check if the variables you need are in the solution step data (for example for the first node)

Reply:
It's already done in the ExecuteInitialize()